modules/oadp-self-service-admin-spec-enforce-nab.adoc

```diff
@@ -0,0 +1,50 @@
+// Module included in the following assemblies:
+//
+// backup_and_restore/application_backup_and_restore/oadp-self-service/oadp-self-service-cluster-admin-use-cases.adoc
+
+:_mod-docs-content-type: CONCEPT
```

Comment:
This seems more like a reference module than a concept module. https://redhat-documentation.github.io/modular-docs/#creating-reference-modules